tests/core/symbolutil.c

```diff
@@ -114,13 +114,13 @@ SYMUTIL_TESTCASE (find_functions_matching)
   g_array_free (functions, TRUE);
 }
 
-static void GUM_CDECL
+void GUM_CDECL
 gum_dummy_function_0 (void)
 {
   g_print ("%s\n", G_STRFUNC);
 }
 
-static void GUM_STDCALL
+void GUM_STDCALL
```

Comment:
This breaks the tests above checking if we can find static functions. Is the problem here that they get inlined? We have a macro to override that, GUM_NOINLINE, e.g.:
https://github.com/frida/frida-gum/blob/4286fa106d4bd752c0af5d39c5f46cf9c3ea73d3/tests/core/interceptor.c#L91